src/Database/migrations/2020_12_10_100036_create_verification_codes_table.php

```diff
@@ -23,7 +23,8 @@ public function up()
             $table->dateTime('verifies_until')->nullable();     // dateTime until the verification is valid (used_at + config value)
             $table->dateTime('used_at')->nullable();            // dateTime when the code has been submitted by verification form
             $table->dateTime('last_touched_at')->nullable();
-            $table->ipAddress('host_ip');
+            $table->ipAddress('ip_address');
```

Comment:
aky je to databazovy typ potom? ako to potom vyzera ulozene v DB? nepoznam ten `ipAddress` helper, ale vyzera zaujimavo

Reply:
> aky je to databazovy typ potom? ako to potom vyzera ulozene v DB? nepoznam ten `ipAddress` helper, ale vyzera zaujimavo

V postgrese sa to ulozi ako typ "inet" - https://www.postgresql.org/docs/9.1/datatype-net-types.html

Reply:
wow, super